file-formats/dsfd/type/zif_aff_dsfd_v1.intf.abap

```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_aff_dsfd_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">CDS Scalar Function Definition</p>
+    "! CDS scalar function definition
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
If you don't support key user tooling for DSDF (yet), I would change the header as well.

```suggestion
      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
```